<launch>

  <param name="robot_description" command="$(find xacro)/xacro.py '$(find pr2_description)/robots/pr2.urdf.xacro'" />

  <node pkg="sbpl_collision_checking" type="test_space" name="sbpl_collision_space_test_hdt" output="screen" respawn="false" >

    <param name="group_name" value="hdt_arm" />

    <param name="dims/x" value="2.0" />
    <param name="dims/y" value="2.0" />
    <param name="dims/z" value="2.0" />

    <param name="origin/x" value="-0.75" />
    <param name="origin/y" value="1.25" />
    <param name="origin/z" value="-0.3" />

    <param name="joint_0" value="joint2_shoulder_pitch" />
    <param name="joint_1" value="joint6_wrist_pitch" />
    <param name="joint_2" value="joint3_arm_roll" />
    <param name="joint_3" value="joint1_shoulder_yaw" />
    <param name="joint_4" value="joint7_wrist_roll" />
    <param name="joint_5" value="joint5_forearm_roll" />
    <param name="joint_6" value="joint4_elbow" />

    <rosparam command="load" file="$(find sbpl_collision_checking)/config/hdt_arm.yaml" />

  </node>

</launch>

